client/src/screens/createTraining/controller.js

```diff
@@ -1,11 +1,34 @@
+export const createSetsPreview = (sets) => {
```

Comment:
controller shouldn't depend on another controller.
I don't know a good place for it right now, so that return it from trainingsApi.
also write a comment above it like `TODO find better place for it`